<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColorDefWidget</class>
 <widget class="QWidget" name="ColorDefWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>170</width>
    <height>584</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="openColorDefFileBtn">
         <property name="text">
          <string>select colorDefFile</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="applyBtn">
         <property name="text">
          <string>apply</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLineEdit" name="regLineEdit"/>
   </item>
   <item>
    <widget class="QListView" name="colorListView"/>
   </item>
   <item>
    <widget class="QTableView" name="colorTableView"/>
   </item>
   <item>
    <widget class="QTableWidget" name="defsTableWidget"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
